src/cards/SpeciesCard.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import EbiSpeciesIcon from 'react-ebi-species'
+
+import renderContentListItems from './renderContentListItems'
+import cardPropTypes from './cardPropTypes'
+
+const SpeciesCard = ({iconSrc, description, content}) =>
+  <div style={{marginBottom:0, paddingBottom: `2rem`, textAlign: `center`}}>
+    {
+      description && description.url ?
+        <a style={{fontSize: `600%`, borderBottom: 0}} href={description.url}>
+          <EbiSpeciesIcon species={iconSrc}/>
+        </a> :
+        <span style={{fontSize: `600%`}}>
+          <EbiSpeciesIcon species={iconSrc}/>
+        </span>
+    }
+
+    {
+      description &&
+      <h5>
+      {
+        description.url ?
+          <a href={description.url}>{description.text}</a> :
+          description.text
+      }
+      </h5>
+    }
+
+    {
+      Array.isArray(content) &&
+      <ul style={{listStyle: `none`}}>
```

Comment:
Set `margin-left: 0` to get rid of the extra space and centre the text properly.

Reply:
Same comment applies to `ExtendableCard`